### PR TITLE
[es] feature: contributing - spanish translation

### PR DESCRIPTION
**Pull request by krol3**
Related the issue: https://github.com/open-telemetry/opentelemetry.io/issues/4887

### CODE REVIEW

**Comment by krol3** [comment]
@ramrodo @theletterf @electrocucaracha, could you review please?

**Review by ramrodo — COMMENTED** [review]
LGTM

File: content/es/docs/contributing/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Contribuir
+description: Aprende cómo contribuir a la documentación de OpenTelemetry.
+weight: 980
+default_lang_commit: 30783526402b69a3ac44eeb0f6cf066732f0bdca
+---
+
+Los colaboradores de la documentación de OpenTelemetry:
```
Comment (theletterf) [inline content/es/docs/contributing/_index.md:8]:
```suggestion
Quienes colaboran para documentar OpenTelemetry:
```

File: content/es/docs/contributing/_index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Mejorar el contenido existente.
+- Crear contenido nuevo.
+- Actualizar el Registro de OpenTelemetry.
+- Mejorar el código que construye el sitio.
```
Comment (theletterf) [inline content/es/docs/contributing/_index.md:13]:
```suggestion
- Mejoran el contenido existente.
- Crean contenido nuevo.
- Actualizan el Registro de OpenTelemetry.
- Mejoran el código que construye el sitio.
```

File: content/es/docs/contributing/_index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Las siguientes guías describen cómo contribuir a la documentación de OpenTelemetry.
+
+Para obtener orientación sobre cómo contribuir al proyecto OpenTelemetry en general, consulta la 
+[Guía del Contribuyente de OpenTelemetry](https://github.com/open-telemetry/community/blob/main/guides/contributor/README.md), que proporciona detalles sobre el Acuerdo de Licencia del Contribuyente y el Código de Conducta. Cada implementación de lenguaje, Collector y convenciones en el [repositorio](https://github.com/open-telemetry/) tiene sus propias guías específicas para contribuir.
```
Comment (theletterf) [inline content/es/docs/contributing/_index.md:18]:
```suggestion
Para obtener orientación sobre cómo contribuir al proyecto OpenTelemetry en general, consulta la 
[Guía del contribuyente de OpenTelemetry](https://github.com/open-telemetry/community/blob/main/guides/contributor/README.md), que proporciona detalles sobre el Acuerdo de Licencia del Contribuyente y el Código de Conducta. Cada implementación de lenguaje, el Collector y las convenciones en el [repositorio](https://github.com/open-telemetry/) tiene sus propias guías específicas para contribuir.
```

**Review by theletterf — CHANGES_REQUESTED** [review]
Nice! Some suggestions.

I'm adding the `default_lang_commit` to all other PRs. To learn what that is about, see https://opentelemetry.io/docs/contributing/localization/#adding-default_lang_commit-to-new-pages

**Comment by theletterf** [comment]
@krol3 PTAL

File: content/es/docs/contributing/_index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+Los colaboradores de la documentación de OpenTelemetry:
+
+- Mejorar el contenido existente.
+- Crear contenido nuevo.
+- Actualizar el Registro de OpenTelemetry.
+- Mejorar el código que construye el sitio.
+
+Las siguientes guías describen cómo contribuir a la documentación de OpenTelemetry.
```
[the remark this answers is not in the document]
Comment (theletterf) [reply content/es/docs/contributing/_index.md:15]: Tilde is required in this case.

**Review by theletterf — APPROVED** [review]

**Pull request merged**